eclipse-scout-core/test/lookup/RestLookupCallSpec.ts

```diff
@@ -262,4 +262,20 @@ describe('RestLookupCall', () => {
     });
   });
 
+  it('aborts calls', () => {
+    let lookupCall = scout.create(SpecRestLookupCall, {
+      session: session,
+      resourceUrl: 'test-api/dummy'
+    });
+    lookupCall.abort();
+
+    lookupCall = scout.create(SpecRestLookupCall, {
```

Comment:
Why do we need to create a new LookupCall?

Reply:
we don't